easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.10.0-ImageExporter.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru pyqtgraph-0.10.0-orig/pyqtgraph/exporters/ImageExporter.py pyqtgraph-0.10.0/pyqtgraph/exporters/ImageExporter.py
```

Comment:
Please add authorship, description and link to bug report (if any) for this patch.

Reply:
@lexming I've just pushed the requested changes.  Let me know if more are needed.